ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_ipahealthcheck.py::TestIpaHealthCheck
         template: *ci-master-latest
         timeout: 3600
```

Comment:
This test needs more time to execute (5400 would be a good value). Please see its definition:
https://github.com/freeipa/freeipa/blob/6033d495d19976708d3ff870523c4c2b9563dfd6/ipatests/prci_definitions/nightly_latest.yaml#L1393-L1403